QuasarHelios2015/src/org/team1540/quasarhelios/AutoEjector.java

```diff
@@ -34,12 +34,18 @@ public static BooleanStatus create() {
     @Override
     protected void autonomousMain() throws AutonomousModeOverException, InterruptedException {
         try {
-            setClampHeight(1.0f);
-            Elevator.setBottom.event();
+            float currentClampHeight = Clamp.height.get();
```

Comment:
You still want ``Clamp.heightReadout`` so you get the current clamp height instead of the desired height.